accepted/build/standard-ci-env-variables.md

```diff
@@ -47,8 +47,25 @@ It is important that these environment variables do not conflict with other vari
 
 * **STANDARD\_CI\_SOURCE\_REVISION\_ID** -- Commit hash / ID; Example: 2ba93796dcf132de447886d4d634414ee8cb069d
 * **STANDARD\_CI\_REPOSITORY\_URL** -- URL for repository; Example: https://github.com/dotnet/corefx
+* **STANDARD\_CI\_REPOSITORY\_URL\_MAP{n}** -- optional mapping of URLs 
 * **STANDARD\_CI\_REPOSITORY\_TYPE** -- Source control manager name; Example: `git`, `TFVC`, `mercurial`, `svn`
 
+### STANDARD_CI_REPOSITORY_URL
+
+The URL to be used to refer to the repository from artifacts produced by the build. If the source control provider supports permanent URLs (i.e. URLs resilient to renames) this should be such permanent URL. 
```

Comment:
I'm confused by this, is `https://github.com/dotnet/corefx` still preferred?
Or should it be `https://raw.githubusercontent.com/dotnet/corefx/master/` (doesn't make sense here imho).

If it's the former then I'm not sure why we need this, renames of a repository should be handled by the hosting provider with proper redirects.

Reply:
It's the former. Redirects work to some extent but not completely. If you rename repo A to B and then repo C to A. Some hosts (VSTS) could handle it by using GUIDs. Something like `https://github.com/{organization-guid}/{repository-guid}`.